packages/element/src/renderElement.ts

```diff
@@ -602,6 +602,9 @@ const generateElementWithCanvas = (
   }
   return prevElementWithCanvas;
 };
+function isEmojiOnly(text: string) {
+  return [...text].every(ch => /\p{Extended_Pictographic}/u.test(ch));
```

Comment:
This matches non-color emojis too, which is overly broad. `Emoji_Presentation` might be a better choice.